src/cmap/connection.ts

```diff
@@ -109,8 +109,10 @@ export interface CommandOptions extends BSONSerializeOptions {
   noResponse?: boolean;
   omitReadPreference?: boolean;
 
-  // FIXME: NODE-2802
-  willRetryWrite?: boolean;
+  // FIXME(NODE-2802): Currently the CommandOptions take a property willRetryWrite which is a hint from executeOperation that the txnNum should be applied to this command.
```

Comment:
incredibly unimportant nit: can we use `TODO` instead of `FIXME`?  Seems like we've been using `TODO` in recent work

Reply:
done

Reply:
done